### DISCUSSION TITLE
I no can use Vercel

### DISCUSSION THREAD

**Discussion by yuyangtian1000** [vercel/community#1828] [Help] [answered]
### Summary

Hi there,

Thank you for getting in touch.

Your account was blocked due to being flagged at registration.

Please let us know what kinds of projects you intend to use the Vercel platform for so we can unblock your account.

Samuel Ko
[▲](https://vercel.com/) Sr. Customer Success Engineer

Visit our [Community](https://github.com/vercel/community/discussions/) for developer-led discussions and help.

i don't know what to do

**Comment by swarnava** [comment, marked as answer]
Hi, you need to reply on the same thread with more details on how you want to use the platform. Once you explain that, our team will unblock your account

**Reply by yuyangtian1000** [reply]
I need to use it to fix chatgpt, for example, I need to access chatgpt API anywhere, but chatgpt can only be used in a specific area.

**Reply by swarnava** [reply]
You need to contact via email. We don't manage registration related question via community :)

**Reply by yuyangtian1000** [reply]
how to do
